indico/modules/events/management/client/js/index.js

```diff
@@ -72,9 +72,13 @@ import {SeriesManagement} from './SeriesManagement';
     containerSelector
   ) {
     const element = document.querySelector(containerSelector);
-    const {eventId, surveyId} = element.dataset;
+    const {eventId, surveyId, surveyActive} = element.dataset;
     ReactDOM.render(
-      <EmailSurveyParticipantsButton eventId={+eventId} surveyId={+surveyId} />,
+      <EmailSurveyParticipantsButton
+        eventId={+eventId}
+        surveyId={+surveyId}
+        surveyActive={JSON.parse(surveyActive)}
```

Comment:
Putting (json) booleans data attributes is kind of uncommon, usually we set the attribute itself conditionally and then use `surveyActive !== undefined` here